docs/sphinxext/mantiddoc/directives/categories.py

```diff
@@ -367,10 +371,37 @@ def create_category_pages(app):
             category_html_path_noext = posixpath.join(document_dir, 'index')
             context['outpath'] = category_html_path_noext + '.html'
             yield (category_html_path_noext, context, template)
-# enddef
 
+    #create a Top level category page
+    # Intitalise the lists
+    all_top_categories = []
+    top_context = {}
+    top_html_path_noext = ""
+    # If the category is a top catergory it will not contain "\\"
+    for top_name, top_category in iteritems(categories):
+        if "\\" not in top_name and top_category not in all_top_categories:
+            all_top_categories.append(top_category)
+
+        template = CATEGORY_PAGE_TEMPLATE
+        top_context["text_page"] = "algorithm_categories.html"
+        top_context["subcategories"] = sorted(all_top_categories, key = lambda x: x.name)
+        #top_html_path_noext = ""
+        # Set up the positions in the toc tree
+        top_category_html_dir = posixpath.join(top_category.name.lower(), 'categories')
+        top_document_dir = posixpath.dirname(top_category_html_dir)
+        top_category_html_path_noext = posixpath.join(top_document_dir, 'index')
+        top_context['outpath'] = "../" + top_category_html_path_noext + '.html'
```

Comment:
Is the `../` required here? Just tried this again and the top level category pages do not work as they miss the top level directory out of the URL.